packages/shorebird_cli/test/src/shorebird_android_artifacts_test.dart

```diff
@@ -0,0 +1,282 @@
+import 'package:path/path.dart' as path;
+import 'package:shorebird_cli/src/shorebird_android_artifacts.dart';
+import 'package:shorebird_cli/src/third_party/flutter_tools/lib/flutter_tools.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group(ShorebirdAndroidArtifacts, () {
+    late Directory projectPath;
+    late ShorebirdAndroidArtifacts shorebirdAndroidArtifacts;
+
+    setUp(() {
+      projectPath = Directory.systemTemp.createTempSync();
+      shorebirdAndroidArtifacts = ShorebirdAndroidArtifacts();
+    });
+
+    test('returns null when cannot find the aab', () {
+      expect(
+        shorebirdAndroidArtifacts.findAppBundle(
+          projectPath: projectPath.path,
+          flavor: null,
+        ),
+        isNull,
+      );
+    });
+    test('returns null when cannot find the apk', () {
+      expect(
+        shorebirdAndroidArtifacts.findApk(
+          projectPath: projectPath.path,
+          flavor: null,
+        ),
+        isNull,
+      );
+    });
+
+    test('find the app bundle without any flavors', () {
+      final artifactPath = path.join(
+        'build',
+        'app',
+        'outputs',
+        'bundle',
+        'release',
+        'app-release.aab',
+      );
+      final artifact = File(
+        path.join(projectPath.path, artifactPath),
+      )..createSync(recursive: true);
+
+      expect(
+        shorebirdAndroidArtifacts.findAppBundle(
+          projectPath: projectPath.path,
+          flavor: null,
+        ),
+        equals(artifact.path),
+      );
+    });
+    test('find the apk bundle without any flavors', () {
```

Comment:
```suggestion
    });

    test('finds the apk without any flavors', () {
```

Also might be nice to group these tests like:

```dart
group('when apk exists', () {
  group('when apk contains flavors', () {
    
  });
  group('when apk does not contain flavors', () {

  });
});

group('when apk does not exist', () {

});
```